SPManager/src/artofillusion/spmanager/SPManagerFrame.java

```diff
@@ -192,19 +189,10 @@ private void doTabbedPaneSelection()
      */
     protected void checkForUpdatedMe()
     {
-	Vector localList = manageSplitPane.getFileSystem().getPlugins();
-	Vector remoteList = updateSplitPane.getFileSystem().getPlugins();
-	SPMObjectInfo localinfo=null, remoteinfo=null;
 
-	for (int i = 0; i < localList.size(); i++) {
-	    localinfo= (SPMObjectInfo) localList.get(i);
-	    if ("SPManager".equals(localinfo.getName())) break;
-	}
-
-	for (int i = 0; i < remoteList.size(); i++) {
-	    remoteinfo = (SPMObjectInfo) remoteList.get(i);
-	    if ("SPManager".equals(remoteinfo.getName())) break;
-	}
+        Predicate<SPMObjectInfo> self = (SPMObjectInfo t) -> t.getName().equals("SPManager");
+        SPMObjectInfo localinfo = manageSplitPane.getFileSystem().getPlugins().stream().filter(self).findFirst().orElse(null);
+        SPMObjectInfo remoteinfo = updateSplitPane.getFileSystem().getPlugins().stream().filter(self).findFirst().orElse(null);
```

Comment:
These lines are far too long. They should be wrapped. Probably align the `.stream()` with `.getPlugins()`